cloud/terraform/aws/installation.md

```diff
@@ -0,0 +1,144 @@
+# Liferay Cloud Native AWS Installation Guide
+
+## Prerequisites
+
+1. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and configure with [IAM credentials](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html).
+2. Install [Terraform CLI](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli).
+3. Install [Helm CLI](https://helm.sh/docs/intro/install/).
+4. Install [kubectl CLI](https://kubernetes.io/docs/tasks/tools/).
+
+## AWS
+
+1. Export your profile for AWS SDK and its tools. 
+
+   ```bash
+   export AWS_PROFILE=[profile]
+   ```
+
+2. Log into AWS CLI.
+
+   ```bash
+   aws sso login
+   ```
+
+## Installation
+
+Clone the repository [TBD].
+
+Once the repository has been cloned, you have two choices:
+
+1. Bring your own AWS account. If you have an AWS account and wish to create a new EKS cluster complete with VPC and networking, follow [EKS Cluster Bootstrap](#eks-cluster-bootstrap).
+
+2. Bring your own EKS cluster. If you have an existing EKS cluster, follow [Liferay Infrastructure Bootstrap](#liferay-infrastructure-bootstrap).
+
+## EKS Cluster Bootstrap
+
+1. Navigate to the `eks` directory. 
+
+2. Edit `terraform.tfvars` to configure your infrastructure. Variables are defined in the `variables.tf` file. By default, the system deploys an EKS cluster in the US West (Oregon) region (us-west-2) spanning two availability zones.
+
+3. Run the following commands:
+
+   ```bash
+   terraform init
+   ```
+ 
+   ```bash
+   terraform apply
+   ```
+
+   You are prompted to apply the changes. 
+
+4. Append the result of `terraform output` to the `../dependencies/terraform.tfvars` file in the `dependencies` directory: 
+
+   ```bash
+   terraform output >> ../dependencies/terraform.tfvars
+   ```
+
+## Liferay Infrastructure Bootstrap
+
+1. Navigate to the `dependencies` directory.
+
+2. Update the `terraform.tfvars` file to configure your infrastructure. Variables are defined in `variables.tf` file. If you followed [EKS Cluster Bootstrap](#eks-cluster-bootstrap), this file is already populated.
+
+3. Run the following commands:
+
+   ```bash
+   terraform init
+   ```
+
+   ```bash
+   terraform apply
+   ```
+
+   You are prompted to apply the changes. 
+
+<!-- Just a sanity check on the above two sections. It sounds like there are two `terraform.tfvars` files in two different directories: `eks` and `dependencies`. In EKS Cluster Bootstrap, you have the reader editing `terraform.tfvars` in `eks`, and then append the result of a command into `../dependencies/terraform.tfvars`. In Liferay Infrastructure Bootstrap, you have the reader edit only `dependencies/terraform.tfvars`, but the instructions (wordsmithed) correspond to the instructions above for the `eks/terraform.tfvars` file. -->
```

Comment:
Nice. Funny enough, I clarified this with @rotty3000 earlier today. We indeed have two `terraform.tfvars` files, [one](https://github.com/brianchandotcom/liferay-portal/pull/160633/files#diff-c768ab1fd598d2994f31582255d14d9cec7140c2ead92268aba28f4654f25b4b) in `dependencies` and the [other](https://github.com/brianchandotcom/liferay-portal/pull/160633/files#diff-469fb110bc24ea9ebe274072054fe564927be29257a9e5be6ba351fe0261dbcc) in `eks`. Step 4 from the EKS Cluster Bootstrap involves appending the `terraform output` result to the `../dependencies/terraform.tfvars` file, so that file should already be populated _if indeed_ the user performed the EKS Cluster Bootstrap procedure.

I believe all is in order, but I'll tag @rotty3000 for another set of eyes.